src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -61,6 +61,22 @@
    {}
    route))
 
+(defn estimated-received-by-chain
+  [{:keys [route token-decimals native-token?]}]
+  (reduce
+   (fn [acc path]
+     (let [estimated-received (:estimated-received path)
+           amount             (money/with-precision
+                               (if native-token?
+                                 (money/wei->ether estimated-received)
+                                 (money/token->unit estimated-received
+                                                    token-decimals))
+                               6)
```

Comment:
We are using this `6` value for limiting decimals in many places, would be great to add a constant for it.

Reply:
done :+1: